types/office-js/index.d.ts

```diff
@@ -19691,6 +19691,44 @@ declare namespace Excel {
          */
         automatic = "Automatic"
     }
+    /**
+     * Represents a command type of `DataConnection`.
+     *
+     * @remarks
+     * [Api set: ExcelApi 1.15]
+     */
+    enum DataSourceType {
+        /**
+         * The data source type is unknown or unsupported.
+         * @remarks
+         * [Api set: ExcelApi 1.15]
+         */
+        unknown = "Unknown",
+        /**
+         * Contains a cube name for an OLAP data source.
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
```

Comment:
@wandyezj Why are there preview APIs in office-js? Here and line 19728

Reply:
(Removing per internal conversation)

Reply:
Those should be removed.